modules/docker-mirror/variables.tf

```diff
@@ -21,13 +21,13 @@ variable "machine_image" {
 
 variable "machine_type" {
   type        = string
-  default     = "n1-standard-2" // 2 vCPU, 7.5GB
+  default     = "n2-standard-2" // 2 vCPU, 8GB
```

Comment:
Much better network bandwidth on this machine, so we use this instead of increasing the number of CPUs on N1 which would have the same effect.